sdc/tests/tests_perf/test_perf_series.py

```diff
@@ -234,14 +235,25 @@ def {func_name}(self):
     ('value_counts', '', [3 * 10 ** 5]),
     ('var', '', [5 * 10 ** 8]),
     ('unique', '', [10 ** 5]),
+    ('isnull', '', [10 ** 7]),
+    ('notna', '', [10 ** 7]),
+    ('var', '', [10 ** 8]),
```

Comment:
`var` added twice

Reply:
It is better to sort lines alphabetically.